CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # [Unreleased](https://github.com/pybamm-team/PyBaMM/)
 
+# [v22.1](https://github.com/pybamm-team/PyBaMM/tree/v22.1) - 2022-01-28
```

Comment:
Could this always make it the last day of the month?

Reply:
Putting the last day on the changelog or running the workflow on the last day?

Reply:
on the changelog